### PR TITLE
Add airport

### PR DESCRIPTION
**Pull request by dennisxu887**
add airport form:
1. addairport.jsp
2. listairports.jsp
3. AirportDao.java
4. AddAirportServlet.java
5. UpdateDeleteAirportServlet.java
6. Airport.java

### CODE REVIEW

File: flightmate/.classpath
Comment (GabrielMontplaisir) [inline flightmate/.classpath:1]: Unsure whether the changes to classpath will break the project.

File: flightmate/src/main/java/com/flightmate/dao/DBUtil.java [hunk 1]
```diff
@@ -20,8 +20,15 @@
 
 public class DBUtil {
 	private static final String USERNAME = "root";
-	private static final String PASSWORD = "";
-	private static String M_CONN_STRING = "jdbc:mysql://localhost/";
+	private static final String PASSWORD = "12345678";
```
Comment (GabrielMontplaisir) [inline flightmate/src/main/java/com/flightmate/dao/DBUtil.java]: It's best if we all use a consistent password. Everyone else left this blank. I suggest you adjust your "root" user to do the same.

File: flightmate/src/main/webapp/dashboard.jsp
Comment (GabrielMontplaisir) [inline flightmate/src/main/webapp/dashboard.jsp:1]: Why did you remove this?

**Review by GabrielMontplaisir — COMMENTED** [review]
I really like the work on Airports. I like that you used a Builder class and it respects the ERD (there's a field missing: "created_at", but that's not such a big deal)

**Review by dennisxu887 — COMMENTED** [review]
I have deleted some files based on your recomendation.
